### PR TITLE
Add hook for 'ismytrainfucked.com'

### PR DESCRIPTION
**Pull request by Cbeck527**
- Bot listens for the text "@bot are the trains fucked?"
- Bot curls http://ismytrainfucked.com and responds with the information
  on the site

### CODE REVIEW

File: modules/ismytrainfucked.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+ # -*- coding: utf8 -*-
+from bs4 import BeautifulSoup
+import requests
+import urllib
+
+import modules
+
+
+@modules.register(rule=r"$@bot are the trains fucked?")
+def ismytrainfucked(bot, msg):
```
Comment (Cbeck527) [inline modules/ismytrainfucked.py]: Actually I don't think `msg` needs to be in there...

File: modules/ismytrainfucked.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+ # -*- coding: utf8 -*-
+from bs4 import BeautifulSoup
+import requests
+import urllib
+
+import modules
+
+
+@modules.register(rule=r"$@bot are the trains fucked?")
```
Comment (orez-) [inline modules/ismytrainfucked.py]: The `?` is being interpreted as regex, and should be escaped. `r"$@bot are the trains fucked\?"`

File: modules/ismytrainfucked.py (hunk 1 above)
Comment (orez-) [inline modules/ismytrainfucked.py]: In my opinion it's good practice to leave the msg in anyway, just to be clear what that parameter WOULD be used for (even though it's never used in this case).

Comment (Cbeck527) [inline modules/ismytrainfucked.py]: :+1:

**Pull request merged**